performance improvements (#142)
AdobeDocs/graphql-mesh-gateway: src/pages/gateway/release-notes.md

The following sections indicate when updates were made to API Mesh for Adobe Developer App Builder. Refer to the [Upgrade version](upgrade.md) for more information on upgrading versions.

## June 29, 2023

This release contains the following changes to API Mesh:

### Bug fixes

> - A previous release introduced an issue that could cause unnecessary delays on Graphql requests. This issue has been resolved and could result in improved performance.

Reviewer: ```suggestion
- A previous release introduced an issue that could cause unnecessary delays on GraphQL requests. This issue has been resolved and could result in improved performance.
```

> - A previous release introduced an issue that could cause unnecessary calls for every query. This issue has been resolved and could result in improved query performance.

Reviewer 2: ```suggestion
- A previous release introduced an issue that could cause unnecessary delays on Graphql requests. This issue has been resolved and could result in improved performance.
```
